charts/wga/templates/unbound/deployment.yaml

```diff
@@ -23,4 +23,11 @@ spec:
           command:
             - /usr/sbin/unbound
             - -dvvv
+        - image: "{{- .Values.global.privateImageRegistry }}/{{- .Values.welcome.image.name -}}:{{- .Values.welcome.image.tag }}"
+          name: wga-welcome
+          imagePullPolicy: {{.Values.welcome.image.pullPolicy }}
+          ports:
+            - containerPort: 80
+              name: welcome
+              protocol: TCP
```

Comment:
This section needs to be referenced in the same service as `unbound` <https://github.com/kraudcloud/wga/blob/0.5.12/charts/wga/templates/unbound/service.yaml> for that port to do anything